### ISSUE TITLE
generated swagger.json fails swagger validation

### ISSUE THREAD

**Issue by CBarraford** [kminami/apib2swagger#15]
Taking the test file https://github.com/kminami/apib2swagger/blob/master/test/input/Attributes.md, and generating a swagger.json file with `apib2swagger`, the json file has errors (if you upload it into http://editor.swagger.io/).

If i convert the file using https://apimatic.io/transformer, the file has no errors (see resulting json below).
```
{
  "swagger": "2.0",
  "info": {
    "version": "",
    "title": "Example API",
    "description": "TODO: Add a description",
    "license": {
      "name": "MIT",
      "url": "http://github.com/gruntjs/grunt/blob/master/LICENSE-MIT"
    }
  },
  "host": "www.example.com",
  "basePath": "/",
  "securityDefinitions": {},
  "schemes": [
    "http"
  ],
  "consumes": [
    "application/json"
  ],
  "produces": [
    "application/json"
  ],
  "paths": {
    "/some/resource": {
      "post": {
        "description": "",
        "operationId": "Create_SomeResource_",
        "produces": [
          "application/json"
        ],
        "parameters": [],
        "responses": {
          "200": {
            "description": ""
          }
        },
        "x-unitTests": [],
        "x-operation-settings": {
          "CollectParameters": false,
          "AllowDynamicQueryParameters": false,
          "AllowDynamicFormParameters": false,
          "IsMultiContentStreaming": false
        }
      }
    }
  }
}
```

**Comment by kminami** [comment]
I tried http://editor.swagger.io/, but got no errors. (It said "Processed with no error")
Could you tell me errors or failed swagger.json?

My converted swagger.json is below.
```
{
    "swagger": "2.0",
    "info": {
        "title": "",
        "version": "",
        "description": ""
    },
    "paths": {
        "/some/resource": {
            "post": {
                "responses": {
                    "200": {
                        "description": "OK",
                        "headers": {},
                        "examples": {}
                    }
                },
                "summary": "",
                "description": "",
                "tags": [],
                "parameters": [
                    {
                        "name": "body",
                        "in": "body",
                        "schema": {
                            "type": "object",
                            "required": [
                                "id"
                            ],
                            "properties": {
                                "id": {
                                    "type": "number"
                                },
                                "name": {
                                    "type": "string"
                                }
                            }
                        }
                    }
                ]
            }
        }
    },
    "definitions": {}
}
```

**Comment by CBarraford** [comment]
Sorry, my bad. Heres a better example.

https://gist.github.com/CBarraford/1082105bbf8a373c5501eac1eb668969

Copying this json into the editor gives errors. Maybe theres something wrong with the way I'm writing my apib doc?

**Comment by kminami** [comment]
Thank you for good example!
I fixed the validation errors at v0.6.2.
